develop-guide/writeblitz/writeblitz.rst

```diff
@@ -246,6 +246,60 @@ other existing apis.
         function: get_traffic_stream_objects
         ...
 
+rest
+____
+
+The ``rest`` action allows you to make rest call to any endpoint on a device. Rest uses http protocols to 
+transfer data. Five http protocols are supported, `get`, `post`, `put`, `patch` and `delete`.
+
+You can find additional information on rest, using this `tutorial
+<https://pubhub.devnetcloud.com/media/genie-feature-browser/docs/#/apis>`_.
+
+.. code-block:: YAML
+
+    test_sections:
+        - plain_actions:
+            - rest:
+                method: get
```

Comment:
better to have all the examples for each method. `post` and `patch` are missing.

Reply:
Added